src/mcts/search.cc

```diff
@@ -187,11 +187,20 @@ void Search::SendMovesStats() const {
   const float parent_q =
       -root_node_->GetQ(0) -
       kFpuReduction * std::sqrt(root_node_->GetVisitedPolicy());
+  const float puct_u_coeff =
+      kCpuct * std::sqrt(std::max(root_node_->GetChildrenVisits(), 1u));
+
   for (Node* child : root_node_->Children()) {
     nodes.emplace_back(child);
   }
   std::sort(nodes.begin(), nodes.end(),
-            [](const Node* a, const Node* b) { return a->GetN() < b->GetN(); });
+            [&parent_q, &puct_u_coeff](const Node* a, const Node* b) {
+              auto an = a->GetN(), bn = b->GetN();
```

Comment:
```cpp
   return std::tie(a->GetN(), a->GetQ(parent_q) + a->GetU() * puct_u_coeff) < 
          std::tie(b->GetN(), b->GetQ(parent_q) + b->GetU() * puct_u_coeff);
```
is more idiomatic, and probably more readable (debatable).

Reply:
Ah yes, I like that better than arbitrarily long chains of comparisons